CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
```

Comment:
Do we want to mention Python 2 and 3.5 deprecation warnings in the changelog? And warn about the impending drop? Eg. 0.15.x or 0.16.x are the final releases for Python 2.7 and 3.5?